### ISSUE TITLE
Feature request: add city and contact form to artist bio

### ISSUE THREAD

**Issue by samnabi** [funmusicplace/mirlo#515]
I'd like to see some more information on artist profiles, namely:

- a contact form for booking requests, etc (this could even just be a link to an email address if we don't want to actually handle sending mail)
- a well-structured city field so we can browse artists by location (keeping in mind there are many cities with the same name across the world, so it shouldn't just be a text field)
    - Maybe something like https://osmnames.org

**Comment by simonv3** [comment]
> a contact form for booking requests, etc (this could even just be a link to an email address if we don't want to actually handle sending mail)

Could this be handled by an artist adding their email address to the links on their profile?

**Comment by samnabi** [comment]
Yes, that could work, although I'd like to see an email link have its own UX. Right now, if I try to add a `mailto:` link then the label shows up as "null".
<img width="1026" alt="Screen Shot 2024-02-09 at 4 45 42 PM" src="https://github.com/funmusicplace/mirlo/assets/1731536/414dde9e-b45e-4f03-9f8f-f65dc9f3e857">

**Comment by Cwpute** [comment]
In addition artists could be gently nudged towards adding any form of contact option on their page.
For example, if they don't have any link set up, they could see a one-time warning or notification saying somehting like "hey you don't seem to have shared any link or way to contact you, this may prevent people from following/contacting you. are you sure you want to proceed ?"

**Comment by simonv3** [comment]
This got kinda specced here https://github.com/funmusicplace/mirlo/discussions/271

**Comment by simonv3** [comment]
@samnabi location fields have been added btw.

**Comment by simonv3** [comment]
So I think the simplest thing to do here for contact form is to add a contact button that does two things: sends a notification so that it appears in the feed and then also sends an e-mail to the user.

**Comment by Cwpute** [comment]
You mean send a notification and email to the person contacted through the button right ?
